server/index.js

```diff
@@ -13,29 +13,75 @@ app.use(cors());
 
 app.use(express.static('public'));
 
+// app.get('/reviews/:id', (req, res) => {
```

Comment:
You may want to consider creating another file instead of commenting out the legacy endpoints.

Reply:
Good call, moved it to another file and adjusted the necessary paths in my seed.